sksurv/tree/tree.py

```diff
@@ -491,6 +514,10 @@ def predict_survival_function(self, X, check_input=True, return_array=False):
         >>> plt.ylim(0, 1)
         >>> plt.show()
         """
+
+        if self.low_memory:
+            raise NotImplementedError("predict_survival_function is not implemented in low memory mode.")
```

Comment:
Can be combined with the check in `predict_cumulative_hazard_function` by adding a separate method `_check_low_memory`.